pkg/lib/ownerutil/util.go

```diff
@@ -398,6 +400,19 @@ func InferGroupVersionKind(obj runtime.Object) error {
 			Version: apiextensionsv1.SchemeGroupVersion.Version,
 			Kind:    "CustomResourceDefinition",
 		})
+	case *policyv1beta1.PodDisruptionBudget:
+		objectKind.SetGroupVersionKind(schema.GroupVersionKind{
+			Group:   policyv1beta1.GroupName,
+			Version: policyv1beta1.SchemeGroupVersion.Version,
+			Kind:    "PodDisruptionBudget",
+		})
+	case *schedulingv1.PriorityClass:
+		objectKind.SetGroupVersionKind(schema.GroupVersionKind{
+			Group:   schedulingv1.GroupName,
+			Version: schedulingv1.SchemeGroupVersion.Version,
+			Kind:    "PriorityClass",
+		})
+		// TODO add GVK for VPA objects
```

Comment:
Should we just include VPAs as well?

Reply:
we could but I was wondering if it was worth importing the whole VPA library as a dependency just for this one additional typemeta check (which is somewhat optional). Just wanted some thoughts around it.

Reply:
If all we're ever going to do on this object at runtime is apply it indiscriminately, then I think ignoring it is fine It seems like that would be the right thing to do given our future move towards arbitrary kube manifests.

But if that is the case, do we need the type check on any of the things we're adding? IMO we should do either all or none. It feels weird to do it for some but specifically not for the one that's an external thing we happen to not vendor.